packages/material-ui/src/OutlinedInput/OutlinedInput.js

```diff
@@ -130,11 +173,7 @@ const OutlinedInput = React.forwardRef(function OutlinedInput(props, ref) {
           }
         />
       )}
-      classes={{
-        ...classes,
-        root: clsx(classes.root, classes.underline),
```

Comment:
You missed the underline classes in the useUtilityClasses above

Reply:
It's been a while since I looked at this but I assumed this was here as an error. This shouldn't have an underline class.